### PR TITLE
Add Covid-19 funding info

### PR DESCRIPTION
**Pull request by mattandrews**
This adds two things (and pairs with [these CMS changes](https://github.com/biglotteryfund/craft-dev/pull/268)):

- A new paragraph showing a specific country's funding status for Covid-19:

image
image
image

- A new "trail text" field for funding programmes which replaces their (longer) intro/description text as it used to be:

image

(this will need to be populated in the CMS)

### CODE REVIEW

File: common/content-api.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ function getFundingProgrammes({
     ]).then((responses) => {
         const [enResults, cyResults] = responses.map(mapAttrs);
         return {
-            meta: head(responses).meta,
+            meta: locale === 'en' ? head(responses).meta : responses[1].meta,
             result: mergeWelshBy('slug')(locale, enResults, cyResults),
         };
```
Comment (davidrapson) [inline common/content-api.js:219]: Wondering if we could rewrite this block to make what `locale === 'en' ? head(responses).meta : responses[1].meta` is doing clearer.  Maybe something like:

```js
const [enResponse, cyResponse] = responses;
return {
    meta: locale === 'en' ? enResponse.meta : cyResponse.meta,
    result: mergeWelshBy('slug')(
        locale,
        enResponse.map(mapAttrs),
        cyResponse.map(mapAttrs)
    ),
};
```

File: controllers/funding/programmes/index.js [hunk 2]
```diff
@@ -48,6 +48,21 @@ router.get('/', injectHeroImage('rosemount-1-letterbox-new'), async function (
...
+            countryCovidStatus = covidStatuses.find(
+                (status) => status.country === locationParam
+            );
+            // Fall back to the UK-wide statement
+            if (!countryCovidStatus) {
+                countryCovidStatus = covidStatuses.find(
+                    (status) => status.country === 'ukWide'
+                );
+            }
+            res.locals.countryCovidStatus = countryCovidStatus;
```
Comment (davidrapson) [inline controllers/funding/programmes/index.js:63]: You might be able to combine this logic like this:

```js
res.locals.countryCovidStatus = covidStatuses.find(function (status) {
    return (
        status.country === locationParam ||
        status.country === 'ukWide'
    );
});
```
Or, possibly:

```js
res.locals.countryCovidStatus = covidStatuses.find(function (
    status
) {
    if (status.country === locationParam) {
        return true;
    } else {
        return status.country === 'ukWide';
    }
});
```

Might not be following this logic though so ignore this if I've oversimplified things

File: common/content-api.js (hunk 1 above)
Comment (mattandrews) [reply common/content-api.js:219]: Good call – needed to be `mapAttrs(enResponse)` but much clearer 👍

(FYI the reason for this change was to allow the `meta` field to return the current locale's version, which is now translateable for the global)

File: controllers/funding/programmes/index.js (hunk 2 above)
Comment (mattandrews) [reply controllers/funding/programmes/index.js:63]: No this is a good shout, used the first approach 👍

**Pull request merged**